Add reference to satellite installation helper (#3766)
theforeman/foreman-documentation: guides/common/modules/snip_red-hat-helper-apps.adoc

ifdef::upgrading-connected,upgrading-disconnected[]

> For interactive upgrade instructions, you can use the {ProjectName} Upgrade Helper on the Red{nbsp}Hat Customer Portal.

Reviewer: Is "interactive upgrade instructions" meant to be "interactive instructions for performing the upgrade" or "instructions for performing an interactive upgrade"? Either way, it would be less ambiguous to spell it out, I think.

Author: The first option is the one we want, updated.

> For interactive upgrade instructions, you can also use the {ProjectName} Upgrade Helper on the Red{nbsp}Hat Customer Portal.

Reviewer 2: Drop `also` here and on line 8?  This implies there is multiple interactive methods.

Author: Good catch, updated.

> This application provides you with an exact guide to match your current version number.

Reviewer: "Exact guide to match your version number" seems like a very odd way to say this. Wouldn't it be better to use something like "...provides upgrade instructions relevant for your current version of {ProjectName}"?

Author: It would be better! Applied (but I'm going with "customized" rather than "relevant").

> You can find instructions that are specific to your upgrade path, as well as steps to prevent known issues.

Reviewer: ```suggestion
As a result, you receive instructions that are specific to your upgrade path, as well as steps to prevent known issues.
```

Author: Applied.

For more information, see https://access.redhat.com/labs/satelliteupgradehelper/[{Project} Upgrade Helper] on the Red{nbsp}Hat Customer Portal.
endif::[]
ifdef::installing-satellite-server-connected,installing-satellite-server-disconnected,installing-capsule-server[]

> For interactive installation instructions, you can use the {ProjectName} Installation Helper on the Red{nbsp}Hat Customer Portal.

Reviewer: Like with the upgrade, consider making less ambiguous - "interactive instructions for performing the installation" or "instructions for performing an interactive installation".

Author: Applied.

> This application provides you with an interactive way to prepare installation instructions to match your required {Project} version number and configuration.

Reviewer: ```suggestion
This application provides you with an interactive way to prepare installation instructions for obtaining your required {Project} version number and configuration.
```

Author: Updated, but not exactly as you suggested. I went with a sentence that uses the word "customized" here too.

For more information, see https://access.redhat.com/labs/rhsih/[{ProjectName} Installation Helper] on the Red{nbsp}Hat Customer Portal.

> endif::[]

Author: This is based on the old lines in con_upgrading-overview.adoc, just copied over to a snippet and duplicated for the new installation app.
